src/style.css

```diff
@@ -1 +1,101 @@
-/* add styles here */
+:root {
+  --color-text: #3D4E61;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  width: 100%;
+  display: flex;
+}
+
+.search-bar--big {
+  margin: 20px 0;
+}
+
+.search__label {
+  width: 100%;
+}
+
+.search__input {
+  cursor: pointer;
+  box-sizing: border-box;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  color: var(--color-text);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+}
+
+.search__input--big {
+  height: 70px;
+  padding: 25px 0 26px 62px;
```

Comment:
you should check these padding values, indent between image loop and input text, background-size, background-position, font line-height. You should get this values directly from figma
So try to pass tests